scripts/run_integration.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+readonly SCRIPT_FOLDER=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+readonly PROJECT_FOLDER="${SCRIPT_FOLDER}/.."
+readonly ARCHIVE_FOLDER=~/archives
```

Comment:
Can't you use /var/cache/cpu_features or /tmp (and change it in .travis.yml also) ?